src/format/span.d

```diff
@@ -345,11 +344,8 @@ final class ListSpan : Span {
 		headerSplit = i;
 	}
 
-	void registerTrailingSplit(size_t i) in {
-		assert(elements.length > 0);
-		assert(elements[$ - 1] <= i);
-		assert(!hasTrailingSplit);
-	} do {
+	void registerTrailingSplit(size_t i) in(elements.length > 0)
+			in(elements[$ - 1] <= i) in(!hasTrailingSplit) {
```

Comment:
It seems like 3+ is where it start to look like crap. Arguably, we could blame the formater, but the fact remains, when there are many of these in line, it's not very readable.

Reply:
reverted